masonite/middleware/SSLRedirectMiddleware.py

```diff
@@ -0,0 +1,26 @@
+"""SSL Redirect Middleware."""
+
+from masonite.request import Request
+
+
+class SSLRedirectMiddleware:
+    """SSL Redirect Middleware."""
+
+    def __init__(self, request: Request):
+        """Inject Any Dependencies From The Service Container.
+
+        Arguments:
+            Request {masonite.request.Request} -- The Masonite request object
+        """
+        self.request = request
+
+    def before(self):
+        """Redirect user to secure URL."""
+        from config import middleware
+        if middleware.SSLRedirect:
+            host = request.environ['HOST']
```

Comment:
@josephmancuso Can I use `APP URL` as a first preference when redirecting the user to `https`?

Reply:
yes